cmd/deploy.go

```diff
@@ -49,9 +49,20 @@ func newDeployCmd(rootConfig *RootCommandConfig) *cobra.Command {
 
 	var deployCmd = &cobra.Command{
 		Use:   "deploy",
-		Short: "Build and deploy your Appsody project to your Kubernetes cluster",
-		Long: `This command extracts the code from your project, builds a local Docker image for deployment,
-generates a deployment manifest (yaml) file if one is not present, and uses it to deploy your image to a Kubernetes cluster, either via the Appsody operator or as a Knative service.`,
+		Short: "Build and deploy your Appsody project to Kubernetes.",
+		Long: `Build and deploy a local container image of your Appsody project to your Kubernetes cluster. 
+		
+The command performs the following steps:
+
+1. Extracts the stack, along with your Appsody project, to a local directory.
```

Comment:
Do we need to specify that the `extract` step happens? Maybe this should instead be specified in the build help text

Reply:
I'll remove this as we already mention the `extract` in the `build` help text. Thanks